vuu/src/main/scala/org/finos/vuu/core/module/typeahead/TypeAheadRpcHandler.scala

```diff
@@ -39,7 +39,7 @@ class TypeAheadRpcHandlerImpl(val tableContainer: TableContainer) extends RpcHan
       case dataTable: DataTable =>
         dataTable.columnForName(column) match {
           case c: Column =>
-            dataTable.primaryKeys.foldLeft(Set[String]())(addUnique(dataTable, c, _, _)).filter(_.startsWith(starts)).toArray.sorted.take(10)
+            dataTable.primaryKeys.foldLeft(Set[String]())(addUnique(dataTable, c, _, _)).filter(_.toLowerCase().startsWith(starts.toLowerCase())).toArray.sorted.take(10)
```

Comment:
I think this probably needs to be removed as it's changing the core behaviour of filtering. Feedback would be appreciated

Reply:
Hi @jjgry, 

Do you mean change the way casing is handled?  So you'd prefer to see it lower cased first before comparison?

Thanks,

Chris

Reply:
Hi Chris,

This is something that I added during development for my own convenience. I don't think this PR is the right place to change this, but it's worth considering the behavior. I think the expected behaviour is probably context-dependent and there is a general lack of filtering consistency in the software I've used. It's possibly worth considering options for the future of filtering in Vuu.

Best, JJ

Reply:
I think we should either have case-insensitive filtering by default or add a flag which allow client to opt into case insensitive filtering. It's fine when a client picks values from a list, but when they are typing to filter that list down, or using starts or ends operators, I think its unfriendly to force them to know the right case

Reply:
especially when fields like descriptions and titles might be inconsistent in their use of case  
in application data

Reply:
I've removed this for now so that this PR can progress and be merged. I agree, Steve, that for starts-with filters case insensitivity could be beneficial for lots of users. I'll leave this thought with you for now